src/Tribe/Importer/File_Importer_Events.php

```diff
@@ -195,27 +196,13 @@ private function build_event_array( $event_id, array $record ) {
 		}
 
 		$cats = $this->get_value_by_key( $record, 'event_category' );
+
 		if ( $this->is_aggregator && ! empty( $this->default_category ) ) {
 			$cats = $cats ? $cats . ',' . $this->default_category : $this->default_category;
 		} elseif ( $category_setting = tribe( 'events-aggregator.settings' )->default_category( 'csv' ) ) {
 			$cats = $cats ? $cats . ',' . $category_setting : $category_setting;
 		}
 
-		if ( $this->is_aggregator ) {
```

Comment:
This looks sensible to me ... but I wonder why it was added in the first place - could there be a reason we need to account for by some other means? (I can't think of one, just posing the question)

Reply:
@barryhughes I definitely hear you and this is worth being careful about.

In inquiring about this and working on this last night, here's what I've found:
- I think the problem is that, while the intended "Show Maps" behavior from tickets like [this one](https://central.tri.be/issues/67228) should apply to most import types,
- It *should not* apply to CSV imports
- In essence, EA needs to leave those poor CSV imports alone
- Leah confirmed this theory [today in Slack here](https://tribe.slack.com/archives/C0258NY1C/p1510244816000625)
- And also, check out the "Notes" in the ticket description of [ticket #67228](https://central.tri.be/issues/67228):
> • CSV imports have a column option that can define this option, so this EA setting is not relevant to CSV imports. There should not be a Show Google Maps setting in the CSV Import Settings section.
>
> • As with the rest of the EA settings, the Global setting impacts imports from all sources (except CSV please, for this setting only) but the per-source settings (again, everything but CSV) can override that global setting.

Reply:
Maybe @bordoni or @borkweb could confirm / weigh-in for some extra certainty here?